src/js/index.js

```diff
@@ -0,0 +1,62 @@
+// step1 - 돔 조작과 이벤트 핸들링으로 메뉴 관리하기.
+// 요구사항1) 메뉴추가
+
+//Enter키 입력시 새로고침 방지
+const espressoMenuForm = document.querySelector('#espresso-menu-form');
+espressoMenuForm.setAttribute('onsubmit', "return false;");
+
+
+const espressoMenuName = document.querySelector('#espresso-menu-name');
+const espressoMenuList = document.querySelector('#espresso-menu-list');
+
+//신규메뉴 추가시 메뉴아이템에 들어갈 html코드 반환 함수
+const espressoMenuItem = (espressoMenuName_value) => {
+    return `<li class="menu-list-item d-flex items-center py-2">
+                <span class="w-100 pl-2 menu-name">${espressoMenuName_value}</span>
+                <button type="button" class="bg-gray-50 text-gray-500 text-sm mr-1 menu-edit-button">수정</button>
+                <button type="button" class="bg-gray-50 text-gray-500 text-sm menu-remove-button">삭제</button>
+            </li>`}
```

Comment:
```suggestion
const espressoMenuItem = (espressoMenuName_value) => {
    return `<li class="menu-list-item d-flex items-center py-2">
                <span class="w-100 pl-2 menu-name">${espressoMenuName_value}</span>
                <button type="button" class="bg-gray-50 text-gray-500 text-sm mr-1 menu-edit-button">수정</button>
                <button type="button" class="bg-gray-50 text-gray-500 text-sm menu-remove-button">삭제</button>
            </li>`
}
```
이렇게 쓰시면 코드가 좀더 깔끔해 보일것같아요

Reply:
} 닫는부분을 줄바꿈 깔끔해보일것 같다는 의미 맞죠!? 반영했습니다 감사합니다!